pkgs/development/python-modules/databricks-connect/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib, isPy37, pkgs, buildPythonPackage, fetchPypi, six, py4j }:
+
+buildPythonPackage rec {
+  pname = "databricks-connect";
+  version = "7.1.0";
+  sha256 = "996a9d0f271f6c7edbd2d85b2efb6ff4e58d15222e80f87ca17fdbf224e17056";
+
+  disabled = !isPy37;
+
+  src = fetchPypi { inherit pname version sha256; };
+
+  sourceRoot = ".";
+
+  propagatedBuildInputs = [ py4j six pkgs.jdk ];
+
+  doCheck = false;
```

Comment:
Can you try and run the tests, or explain why they don't work?

Reply:
When running test get the following error. Have to look more in to it. 
```
running test
WARNING: Testing via this command is deprecated and will be removed in a future version. Users looking for a generic test entry point independent of test runner are encouraged to use tox.
running egg_info
writing databricks_connect.egg-info/PKG-INFO
writing dependency_links to databricks_connect.egg-info/dependency_links.txt
writing entry points to databricks_connect.egg-info/entry_points.txt
writing requirements to databricks_connect.egg-info/requires.txt
writing top-level names to databricks_connect.egg-info/top_level.txt
package init file 'deps/bin/__init__.py' not found (or not a regular file)
package init file 'deps/sbin/__init__.py' not found (or not a regular file)
package init file 'deps/jars/__init__.py' not found (or not a regular file)
package init file 'pyspark/python/pyspark/__init__.py' not found (or not a regular file)
package init file 'lib/__init__.py' not found (or not a regular file)
package init file 'deps/data/__init__.py' not found (or not a regular file)
package init file 'deps/licenses/__init__.py' not found (or not a regular file)
package init file 'deps/conf/__init__.py' not found (or not a regular file)
package init file 'deps/examples/__init__.py' not found (or not a regular file)
reading manifest file 'databricks_connect.egg-info/SOURCES.txt'
reading manifest template 'MANIFEST.in'
warning: no previously-included files matching '*.py[cod]' found anywhere in distribution
warning: no previously-included files matching '__pycache__' found anywhere in distribution
warning: no previously-included files matching '.DS_Store' found anywhere in distribution
writing manifest file 'databricks_connect.egg-info/SOURCES.txt'
running build_ext
Could not find valid SPARK_HOME while searching ['/', '/build/pyspark', '/build/pyspark', '/']
builder for '/nix/store/pmpx14b0j3zlggdf3kzndf76jdpvqxy1-python3.8-databricks-connect-7.1.0.drv' failed with exit code 255
error: build of '/nix/store/pmpx14b0j3zlggdf3kzndf76jdpvqxy1-python3.8-databricks-connect-7.1.0.drv' failed
```

Reply:
Have no take a close look. There are no automatic tests in the package that I can find. 

But after package is installed there is test command `databricks-connect test`, for it to work you have to configure connection to a databricks cluster with `databricks-connect configure`. So user have to run test manually after install. 

Have added a comment about this in nix expression.

Reply:
I noticed it complaining about a missing SPARK_HOME in that console output, do you need to add a dependency on Java Spark?